packages/runtime-node/test/node-environments-manager.unit.ts

```diff
@@ -15,18 +23,27 @@ chai.use(chaiAsPromised);
 
 const runFeatureOptions = { featureName: 'engine-node/x' };
 
+const env = new Environment('dev', 'node', 'single');
 const comEntry: IStaticFeatureDefinition = {
     filePath: require.resolve('@wixc3/engine-core/dist/communication.feature'),
     packageName: '@wixc3/engine-core',
     scopedName: 'engine-core/communication',
 };
 
+const server2Env = new Environment('server-two', 'node', 'single');
+
 const engineNodeEntry: IStaticFeatureDefinition = {
     dependencies: [comEntry.scopedName],
     envFilePaths: {
         server: require.resolve('@fixture/engine-node/dist/feature/x.server.env'),
     },
-    exportedEnvs: [{ name: 'server', type: 'node' }],
+    exportedEnvs: [
+        {
+            name: 'server',
+            type: 'node',
+            env: serverEnv,
```

Comment:
if we now enforce passing envs, and not strings, why don't we pass only the env instances? in this case there is a mismatch between the serverEnv.name and the value in the key "name", which might cause us issues down the line